prompt.js

```diff
@@ -48,6 +47,16 @@ module.exports = {
       ],
       default: "none",
     },
+    {
+      name: "testing",
+      message: "Testing Framework:",
+      type: "select",
+      choices: [
+        { message: "None", name: "none" },
+        { message: "Jest", name: "jest" },
```

Comment:
jest'i test runner altına alabiliriz, testing framework olarak react-testint-library olabilir

Reply:
aslinda cok yetersiz sayilmaz `react-test-renderer`'i de ekleyebiliriz framework icin. enzyme'in react 17 yle ilgili problemleri vardi duzeldiyse onu da ekleyelim :D

Reply:
bir de, test framework'ü önce sormamız lazım. Testing library veya herhangi bir şey seçilirse jest mutlaka kurulmalı. tek soruluk bir seçenek olarak çıkabilir